<ui version="4.0" >
 <class>BisquikWebAccessDialog</class>
 <widget class="QDialog" name="BisquikWebAccessDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bisquik Access</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Bisque server configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Url</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="urlEdit" >
        <property name="text" >
         <string>http://dough.ece.ucsb.edu/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="userEdit" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="1" column="2" >
       <widget class="QPushButton" name="searchButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="passwordEdit" >
        <property name="maximumSize" >
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Bisque web</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="1" column="0" >
       <widget class="QWebView" name="webView" >
        <property name="url" >
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QWidget" native="1" name="widget_2" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>4</number>
         </property>
         <property name="margin" >
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="backButton" >
           <property name="text" >
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton" >
           <property name="text" >
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="value" >
            <number>0</number>
           </property>
           <property name="format" >
            <string>%p%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="progressLabel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="pathLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor" >
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoLoadCheck" >
        <property name="text" >
         <string>Auto load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons" >
         <set>QDialogButtonBox::Close|QDialogButtonBox::Open</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>urlEdit</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>webView</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>autoLoadCheck</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
